### ISSUE TITLE
RSSGuard 3.6.1 and 3.7 error: Owncloud Server running an unsupported version.

### ISSUE THREAD

**Issue by RBauglir** [martinrotter/rssguard#255]
I'm running:
- RSSGuard 3.6.1 (also tested 3.7 with the same result).
- Nextcloud 18.0.7 (previously 18.0.6 with the same result).
- News plugin 14.0.11.

The problem is:
- RSSGuard doesn't synchronize, giving a message, when testing the configuration, like (I'm translating) "Selected Owncloud News server is running an unsupported version. At least version 6.0.5 is required".

Thank you for your work.

**Comment by martinrotter** [comment]
Do you have by chance a testing account on that nextcloud server available for me to test?

**Comment by RBauglir** [comment]
> Do you have by chance a testing account on that nextcloud server available for me to test?

You've got mail.

**Comment by martinrotter** [comment]
I did some testing, the error message should give you "version" if Nextcloud it found, can you please run RSS Guard in english localization, reproduce the issue and paste the exact message you get?

**Comment by martinrotter** [comment]
Also I pushed some commit to see debugging output from relevant piece of code. Can you please test development build from commit a3d1420? It should be compiled in 25 minutes, simply download newest file from [here](https://bintray.com/martinrotter/rssguard/Development/Windows#files/) (sort from newest files).

**Comment by RBauglir** [comment]
Hi, Martin. I've being testing too and have found that it was my problem: there was something corrupt in my "RSS Guard" folder (RSSGuard wasn't detecting News plugin version), and all is right again after regenerating it. I intended to delete that folder when testing version 3.7, but I'm afraid I searched for "rssguard" and "RSSGuard", but not for "RSS Guard", and as I didn't find it I thought I'd already erased it.

I'm sorry for wasting you time. Thank you so much for your help.

Anyway, tell me if you still want me run any tests.
